src/features/student/views/StudentEducationActivitiesView/components/StudentDetailedActivityCard/StudentDetailedActivityCard.vue

```diff
@@ -0,0 +1,260 @@
+<script lang="ts" setup>
+import type { AvLocale } from '@/types/i18n.types'
+import { formatDateToLocaleString } from '@/common/utils'
+import { studentActivityRoute } from '@/features/student/routes'
+import { type ActivityDTO, ActivityStatus } from '@/types'
```

Comment:
Hello Pascal,

C'est pas iso avec la payload discuté durant sprint planning : 

https://github.com/avenirs-esr/AVENIRS-Project/issues/275
```suggestion
import { type AmsViewDTO, AmsStatus } from '@/types'
```